src/components/security/security-controller.ts

```diff
@@ -4,20 +4,61 @@ import { PATH_DATA } from "../../app.constants";
 import { hasAllowedRSAServices } from "../../middleware/check-allowed-services-list";
 
 export async function securityGet(req: Request, res: Response): Promise<void> {
-  const { email, phoneNumber, isPhoneNumberVerified } = req.session.user;
-  const phoneNumberLastFourDigits = phoneNumber ? phoneNumber.slice(-4) : null;
+  const { email } = req.session.user;
 
   const supportActivityLogFlag = supportActivityLog();
+
   const hasHmrc = await hasAllowedRSAServices(req, res);
 
   const activityLogUrl = PATH_DATA.SIGN_IN_HISTORY.url;
 
+  const mfaMethods = (req.session.mfaMethods || []).map((method) => {
+    let key: string,
+      value: string,
+      actions = {};
+
+    if (method.mfaMethodType === "SMS") {
+      const phoneNumber = method.endPoint ? method.endPoint.slice(-4) : null;
+      key = req.t("pages.security.mfaSection.summaryList.phoneNumber.title");
+      value = req
+        .t("pages.security.mfaSection.summaryList.phoneNumber.value")
+        .replace("[phoneNumber]", phoneNumber);
+      actions = {
```

Comment:
Once we have the functionality to manage the auth apps, we can refactor this a bit.